scripts/upload.sh

```diff
@@ -35,13 +35,7 @@ s3_upload()
     echo ""
     verbose_item "Uploading '$1'"
     # aws s3 cp "$DRY_RUN" --region us-west-1 "$1" "$REMOTE_PATH/$1"
```

Comment:
I think this comment does not makes sense anymore; do you know what it used to do?